script/console

```diff
@@ -10,7 +10,7 @@ OptionParser.new do |opt|
 end
 
 libs =  " -r irb/completion"
-libs << %( -r ubygems)
+libs << %( -r rubygems)
```

Comment:
This should just be removed. Rubygems is always loaded nowadays.

Reply:
✅